react/todo-app-fe/src/components/Login.js

```diff
@@ -0,0 +1,60 @@
+import React, {useState} from 'react';
+import {Col, Form, Button, Card} from 'react-bootstrap';
+import {Link} from 'react-router-dom';
+
+export default function Login() {
+
+    const [logData, setLogData] = useState({
+        'username': '',
+        'password':''
+    });
+
+    const loginUser = () => {
+        console.log('clicked');
+    }
+
+    return(
+        <div className="Login">
+            <Form onSubmit={loginUser} as={Col} className="col-md-4 mr-auto ml-auto mt-5 shadow-lg pl-5 pr-5 rounded pt-5 pb-5" style={{'backgroundColor':'rgb(245,247,248)'}}>
+                        <h3 className="text-left mb-4">Log In</h3>
+                        
+                        <Form.Row  className="text-left">
+                            <Form.Group as={Col} controlId="formGridUserName">
+                            <Form.Label>Username</Form.Label>
```

Comment:
Tekstove obicno uvek lokalizujemo ali nemoj to sada, cisto da znas za posle kada budemo radili imdb